advanced/Scripts/webpage.sh

```diff
@@ -110,7 +110,8 @@ SetWebPassword() {
         # Prevents a bug if the user presses Ctrl+C and it continues to hide the text typed.
         # So we reset the terminal via stty if the user does press Ctrl+C
         trap '{ echo -e "\nNo password will be set" ; stty sane ; exit 1; }' INT
-        read -s -p -r "Enter New Password (Blank for no password): " PASSWORD
+        # shellcheck disable=SC2162
+        read -s -p "Enter New Password (Blank for no password): " PASSWORD
```

Comment:
The `-r` can still be used, the order of the flags was causing some issues, `-p` needs to be followed by the actual prompt text and in the existing it was followed by `-r` causing issues. 

```suggestion
        read -s -r -p "Enter New Password (Blank for no password): " PASSWORD
```

Reply:
And also needs to be `read -r -a <arrayname>` for the other and that will resolve the `read-only` error.